ex2-tema2.js

```diff
@@ -0,0 +1,26 @@
+//Calculați suma numerelor dintr-un șir de numere
```

Comment:
```suggestion
// Calculați suma numerelor dintr-un șir de numere
```
Eventual, se recomanda de lasat un spatiu dupa cele 2 `//`